### ISSUE TITLE
Species.tau file has NAs

### ISSUE THREAD

**Issue by C20H25N30** [kfuku52/amalgkit#85]
After a amalgkit run with 3 samples excluded, the .tau output file gives a "highest" and "order", but the numerical tau value is NA. 

Example below. 

<html>
<head>
	
	
	<title></title>
	<meta name="generator" content="LibreOffice 7.1.6.2 (Windows)"/>
	<style type="text/css">
		body,div,table,thead,tbody,tfoot,tr,th,td,p { font-family:"Liberation Sans"; font-size:x-small }
		a.comment-indicator:hover + comment { background:#ffd; position:absolute; display:block; border:1px solid black; padding:0.5em;  } 
		a.comment-indicator { background:red; display:inline-block; border:1px solid black; width:0.5em; height:0.5em;  } 
		comment { display:none;  } 
	</style>
	
</head>

<body>

tau | highest | order |  
-- | -- | -- | --
lcl\|NC_037638.1_cds_XP_623975.1_1 | NA | embryo_W | embryo_W\|ovary_W\|midgut_W\|embryo_M\|hypopharyngeal_glands_W\|larval_gut_W\|mandibular_gland_W\|nasonov_gland_W\|skeletal_muscle_W\|malpighian_tubule_W\|adipose_Q\|head_and_thorax_Q\|adipose_W\|sting_gland_W\|antennae_W\|second_thoracic_ganglia_W\|brain_Q\|mushroom_bodies_M\|brain_W\|mandibular_gland_Q\|mushroom_bodies_W
lcl\|NC_037638.1_cds_XP_623952.2_2 | NA | embryo_M | embryo_M\|ovary_W\|nasonov_gland_W\|malpighian_tubule_W\|midgut_W\|adipose_Q\|mandibular_gland_W\|hypopharyngeal_glands_W\|antennae_W\|sting_gland_W\|brain_Q\|second_thoracic_ganglia_W\|larval_gut_W\|skeletal_muscle_W\|mushroom_bodies_M\|brain_W\|mushroom_bodies_W\|embryo_W\|adipose_W\|mandibular_gland_Q\|head_and_thorax_Q
lcl\|NC_037638.1_cds_XP_006557469.1_3 | NA | embryo_M | embryo_M\|embryo_W\|antennae_W\|midgut_W\|second_thoracic_ganglia_W\|nasonov_gland_W\|hypopharyngeal_glands_W\|skeletal_muscle_W\|mandibular_gland_W\|malpighian_tubule_W\|adipose_Q\|sting_gland_W\|larval_gut_W\|mushroom_bodies_M\|ovary_W\|brain_Q\|brain_W\|mandibular_gland_Q\|head_and_thorax_Q\|adipose_W\|mushroom_bodies_W
lcl\|NC_037638.1_cds_XP_006557467.1_4 | NA | embryo_M | embryo_M\|embryo_W\|midgut_W\|hypopharyngeal_glands_W\|mandibular_gland_Q\|larval_gut_W\|brain_W\|mushroom_bodies_W\|adipose_W\|sting_gland_W\|head_and_thorax_Q\|ovary_W\|second_thoracic_ganglia_W\|mushroom_bodies_M\|malpighian_tubule_W\|nasonov_gland_W\|adipose_Q\|mandibular_gland_W\|skeletal_muscle_W\|antennae_W\|brain_Q

</body>

</html>

**Comment by Hego-CCTB** [comment]
Did amalgkit finish without errors?
Do other amalgkit outputs of this run OK, or do they have problems as well?

**Comment by C20H25N30** [comment]
Amalgkit finished with no errors. 

Yes I have done other runs where the .tau calculations were numerical and complete.

**Comment by Hego-CCTB** [comment]
How about the species.curate_group.mean.tsv ? Do they look "normal" (like, all tissues there, values reasonable, etc.)?

**Comment by Hego-CCTB** [comment]
@C20H25N30 I think I found the cause of this, but I'm still investigating where and why it happens. This is specific to your dataset, so let's continue this discussion via mail.

**Comment by Hego-CCTB** [comment]
This issue was caused by all samples of a certain curate_group (i.e. tissue) being marked for exclusion, which caused problems with the tau calculation further down the line.

I have added a failsave that works like this:
1. print warning message that all samples of a tissue have been marked for exclusion
2. continue the algorithm without this tissue

updated in amalgkit ver. 0.5.1:
https://github.com/kfuku52/amalgkit/commit/0bf4fc3083336883b3aab8aed87b1f1f0e9dba99
